\documentclass{article}

\usepackage[margin=0.5in]{geometry}
\usepackage{graphicx}
\usepackage{pdflscape}
\usepackage[labelformat=empty]{caption}

\pagenumbering{gobble}

\begin{document}

%% Figure 1

\begin{landscape}
  \begin{figure}
    \begin{center}
      \includegraphics[width=10in]{figures/Fig1.pdf}
    \end{center}
    \caption*{\textbf{Fig. 1: a,} Overview of the machine learning framework used to establish potential associations between gene lists and Alzheimer’s Disease. (i) The framework accepts as input gene lists derived from experimental data or extracted from database resources or literature. (ii) Given a gene expression matrix, the framework subsamples it to a particular gene list of interest, and (iii) subsequently trains and evaluates through cross-validation a predictor of Braak stage of disease. (iv) The process is repeated for randomly-selected gene lists of equal lengths to determine whether predictor performance associated with the gene list of interest is significantly higher than what’s expected by chance. \textbf{b,} AMP-AD datasets used by the machine learning framework. The three datasets used to evaluate the predictive power of gene lists are provided by The Religious Orders Study and Memory and Aging Project (ROSMAP), The Mayo Clinic Brain Bank (MAYO) and The Mount Sinai/JJ Peters VA Medical Center Brain Bank (MSBB). The schematic highlights regions of the brain that are represented in each dataset. The MSBB dataset spans four distinct regions, which are designated using Brodmann (BM) area codes. \textbf{c,} Performance of predictors trained on gene lists reported in previous studies of AMP-AD datasets. The predictors are evaluated for their ability to distinguish early-vs-late disease stages with performance reported as area under the ROC curve (AUC). The vertical line on each row denotes predictor performance associated with a gene list reported in the literature, while the background distribution is constructed over randomly-selected lists of matching lengths. Each row is annotated with the pubmed ID of the study, the supplemental resource that contained the gene list, and a short keyphrase providing functional context.}
  \end{figure}
\end{landscape}

\newpage

%% Figure 2

\begin{figure}
  \includegraphics[width=7.5in]{figures/Fig2.pdf}
  \caption*{\textbf{Fig. 2: a,} Overview of the 3’ DGE experimental protocol used to derive drug-associated gene expression signatures. ReNcell VM human neural progenitor cells were plated and differentiated for 10 days, resulting in a mixed cell population of neurons, glia and oligodendrocytes. The mixed culture was subsequently treated with a panel of drugs (Table 1) at 10 µM for 24 h and frozen in a lysis buffer until library preparation. RNA was extracted and reverse transcribed into cDNA in each well of the plate, followed by pooling and preparation of mRNA libraries. After sequencing, mRNA reads were demultiplexed according to well barcodes, and the resulting gene expression profiles were processed by a standard differential expression method to derive drug-associated gene lists. \textbf{b,} A highlight of two compounds whose gene lists consistently yield improved performance over the randomly-selected lists of equal length. Shown is performance associated with predicting early-vs-late disease stages in several AMP-AD datasets. Each row corresponds to an evaluation of gene lists in a single dataset; MSBB evaluation is subdivided into four brain regions, specified as Brodmann Area. The vertical line denotes performance of the drug-associated list, while the background distribution shows performance of gene lists randomly selected from the same dataset. The drugs are annotated with their nominal targets.}
\end{figure}

%% Figure 3

\begin{landscape}
  \begin{figure}
    \begin{center}
      \includegraphics[width=9.5in]{figures/Fig3.pdf}
    \end{center}
    \caption*{\textbf{Fig. 3: Top 15 FDA-approved (left) and experimental/investigational (right) drugs, sorted by harmonic mean p-value.} Each heatmap shows p-values associated with a drug’s predictive performance across two AMP-AD datasets, ROSMAP and MSBB. The MSBB analysis is further subdivided by the brain region, specified as Brodmann Area. The last column in each heatmap shows the harmonic mean p-value (HMP). The rows are annotated with the name of the drug/compound, its nominal target, and the index of the corresponding DGE experiment. Additional annotations include information about each compound’s approval status (approved / investigational / experimental) and whether compounds were found to be toxic in neuronal cell cultures.}
\end{figure}
\end{landscape}

%% Figure 4

\begin{figure}
  \centering
  \includegraphics[width=7in]{figures/Fig4.pdf}
\end{figure}

\begin{figure}
  \caption{\textbf{Fig. 4: a,} Overview of Target Affinity Spectrum (TAS) score computation from raw drug binding data. Three types of drug binding data were sourced from ChEMBL and from the internal Laboratory Systems of Pharmacology dataset that have not yet been incorporated into ChEMBL. Empirically derived thresholds for the different data types where used to assign TAS scores to each drug-target pair. Multiple measurements for the same drug-target combination were aggregated along the first quartile to define the final TAS value. \textbf{b,} Binding affinity of compounds in the ranked list to every member of the Janus Kinase family. The compounds are sorted in increasing order by the harmonic mean p-value (as defined in Figure 3) along the x-axis. The top heatmap shows the binding affinity of each compound to the selected targets, explicitly naming the FDA-approved drugs. Colored and gray tiles denote confirmed binders and non-binders, respectively; missing entries correspond to unknown affinity values. The combined affinity is defined as the strongest binding (lowest TAS score) among all four JAK targets. The bottom plot shows the breakdown of the combined affinity values by TAS-specific empirical cumulative distribution functions (ecdfs). Each line shows ecdfs for all drugs that bind the corresponding target with a TAS score of 1 (dark orange), 2 (orange) or 3 (light orange). \textbf{c,} Top targets whose binding affinity correlates most strongly with the compound ranking. The ecdfs of confirmed non-binders (TAS = 10) are shown as gray dashed lines for reference. Area under ecdf can be interpreted as a summary statistic that captures the position of drugs binding to that target with the corresponding affinity in the ranked list. Correlation between the drug ranking and TAS values was computed using Kendall’s Tau test, with the associated p value displayed in the bottom right corner of each plot.}
\end{figure}

%% Figure 5

\begin{landscape}
  \begin{figure}
    \centering
    \includegraphics[width=9.5in]{figures/Fig5.pdf}
  \end{figure}
\end{landscape}

\begin{figure}
  \caption{\textbf{Fig. 5: a,} An example polypharmacology test with a focus on RPS6KA1 and TYK2. The drugs are ranked by the harmonic mean p-value (as in Figures 3 and 4), and the distributions of drugs bindings to both RPS6KA1 and TYK2 (left), those binding to RPS6KA1 but not TYK2 (middle) and, conversely, TYK2 but not RPS6KA1 (right) are shown along this ranking. Individual drugs that bind those targets are annotated by vertical tick marks directly below the corresponding distribution. \textbf{b,} Top 10 synergistic and top 10 antagonistic relationships between pairs of targets. The distributions in each plot are compared using Wilcoxon Rank Sum test, with the resulting p value presented in the bottom right corner. If compounds that bind both targets appear significantly closer to the top of the ranked list (left side of the x axis), we define the target pair to be synergistic. Conversely, a pair of targets with an explicit non-binding interaction observed among the top-ranking compounds is defined to be antagonistic. A set of five neutral target pairs (i.e., no significant synergistic or antagonistic effect) is included for reference.}
\end{figure}

\end{document}
